main.tf

```
@@ -23,4 +23,5 @@ resource "aws_cloudtrail" "default" {
   event_selector                = "${var.event_selector}"
   kms_key_id                    = "${var.kms_key_arn}"
   is_organization_trail         = "${var.is_organization_trail}"
+  sns_topic_name                = "${var.enable_sns == "true" ? var.sns_topic_name : ""}"
```

```suggestion
  sns_topic_name                = "${var.sns_topic_name}"
```
why not just use `sns_topic_name`? since we are going to pass empty string in case of not `enable_sns`